orc8r/cloud/go/obsidian/tests/unit_test_utils.go

```diff
@@ -134,6 +134,18 @@ func (j *jsonMarshaler) MarshalBinary() (data []byte, err error) {
 	return json.Marshal(j.v)
 }
 
+func StringMarshaler(s string) encoding.BinaryMarshaler {
+	return &stringMarshaler{s}
+}
+
+type stringMarshaler struct {
+	s string
+}
+
+func (m *stringMarshaler) MarshalBinary() (data []byte, err error) {
+	return []byte(m.s), nil
+}
```

Comment:
@hcgatewood Very similar situation to the last PR but it doesn't seem like we have the luxury to add a MarshalBinary() method to the template in this case. For now, it seems like we would inevitably have to add a StringMarshaler but I would love to know if there is a more efficient solution.

Reply:
Yeah this is fine. Since you can't create methods for builtin types, we'd have to wrap the string in some object one way or another, and this is a simple way to handle it